### PR TITLE
Web3 1188 remove prepare script for package publish

### PR DESCRIPTION
**Pull request by rushby**

### CODE REVIEW

File: .github/workflows/ci-publish-package.yml [hunk 1]
```diff
@@ -69,11 +65,19 @@ jobs:
...
+
       - name: Authenticate to npm registry
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: echo "//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}" > ~/.npmrc
 
+      - name: Remove prepare script (Husky)
+        run: |
+          node -e "let pkg = require('./package.json'); delete pkg.scripts.prepare; require('fs').writeFileSync('./package.json', JSON.stringify(pkg, null, 2));"
+
```
Comment (yurikoinaba) [inline .github/workflows/ci-publish-package.yml:80]: As discussed, let's try consolidating this into one job.
Then use the conditions `if: ${{ success() }}` and `if: steps.run_tests.outcome == 'success'` for the publish steps

Then I don't think the prepare script will be an issue bc `husky` is installed in a previous step

Comment (yurikoinaba) [reply .github/workflows/ci-publish-package.yml:80]: And just want to double check that you tried `HUSKY=0 npm publish`

Comment (yurikoinaba) [reply .github/workflows/ci-publish-package.yml:80]: It also seems standard practice to do a `npm ci` before `npm publish`.
That might be best and simplest.

https://github.com/HorizenLabs/zkverifyjs/new/main?filename=.github%2Fworkflows%2Fnpm-publish.yml&workflow_template=ci%2Fnpm-publish

This is the example from GitHub https://github.com/HorizenLabs/zkverifyjs/actions/new?category=none&query=npm+publish

**Review by yurikoinaba — APPROVED** [review]

File: .github/workflows/ci-publish-package.yml [hunk 2]
```diff
@@ -36,39 +36,13 @@ jobs:
         run: npm run build
 
       - name: Run tests
+        id: run_tests
```
Comment (yurikoinaba) [inline .github/workflows/ci-publish-package.yml:39]: This isn't used

Comment (rushby) [reply .github/workflows/ci-publish-package.yml]: Removed

**Pull request merged**